index.test-d.ts

```diff
@@ -0,0 +1,23 @@
+import { expectType, expectAssignable, expectError } from "tsd"
```

Comment:
Oh my god, I missed this library completely... I'm losing my edge

Reply:
🤯